.config/mise.toml

```diff
@@ -0,0 +1,95 @@
+#:schema https://mise.jdx.dev/schema/mise.json
+
+[vars]
+bacon_version = "3.16.0"
+cargo_insta_version = "1.43.1"
+cargo_nextest_version = "0.9.103"
+rust_version = "1.85"
+uv_version = "0.8.13"
+
+[tools]
+# We need cargo-binstall so that Mise would download "cargo:" tools instead of building them.
+"cargo-binstall" = "latest"
+
+[tasks."build:cli-reference"]
+description = "Build the command line reference (cli/tests/cli-reference@.md.snap)"
+tools.rust = "{{vars.rust_version}}"
+tools."cargo:cargo-insta" = "{{vars.cargo_insta_version}}"
+run = "cargo insta test --accept --workspace -- test_generate_md_cli_help"
+
+[tasks."build:debug"]
+alias = "Build jj in debug mode"
+description = "Build jj"
+tools.rust = "{{vars.rust_version}}"
+run = "cargo build"
+
+[tasks."build:docs"]
+description = "Build documentation into rendered-docs/ for offline use"
+tools.uv = "{{vars.uv_version}}"
+run = "uv run mkdocs build"
+
+[tasks."build:release"]
+description = "Build jj in release mode"
+tools.rust = "{{vars.rust_version}}"
+run = "cargo build --release"
+
+
+[tasks."check:clippy"]
+description = "Lint the code with Clippy"
+tools.rust = "{{vars.rust_version}}"
+run = "cargo clippy --workspace --all-targets"
+
+[tasks."check:format"]
+description = "Check the code format"
+tools.rust = "nightly"
+run = "cargo fmt --check"
+
+[tasks."check:test"]
```

Comment:
(Beyond the scope of this PR, no need to add it here) I'm curious, does mise allow tasks to take arguments? I often want to run a subset of tests, so if I could do something like `mise run check:test test_squash` to run something like `cargo insta test  --workspace --test-runner nextest -- test_squash`, that could be nice.

Reply:
`mise` supports passing task parameters. I tested it with `cargo nextest` instead of `cargo insta test`… but `cargo insta test` requires `--` at the end of the command. I've added it.

Also note that mise allows to not use `run` when the task name doesn't collide with a mise command, and that I've added a `test` alias, because running the tests is very common. So we can run `mise test squash` for example.